apps/src/lib/node/ledger/shell/finalize_block.rs

```diff
@@ -497,6 +497,9 @@ where
                 },
             )
             .expect("Must be able to update validator sets");
+        // send the latest oracle configs. These may have changed due to
+        // governance.
+        self.update_eth_oracle();
```

Comment:
I would add this under `if new_epoch {` (in `finalize_block()`). `update_epoch()`, as the docstring suggests, is only for validator set and consensus param changes